src/palace/manager/scripts/playtime_entries.py

```diff
@@ -179,76 +188,114 @@ def do_run(self):
             Configuration.REPORTING_NAME_ENVIRONMENT_VARIABLE, ""
         )
 
-        # format report name for use in csv attachment filename below
-        subject_prefix = reporting_name
-        if len(reporting_name) > 0:
-            subject_prefix += ": "
-
-        email_subject = f"{subject_prefix}Playtime Summaries {formatted_start_date} - {formatted_until_date}"
-        reporting_name_with_no_spaces = reporting_name.replace(" ", "_") + "-"
         link_extension = "csv"
         uid = uuid_encode(uuid.uuid4())
-        linked_file_name = (
-            f"playtime-summary-{reporting_name_with_no_spaces}"
-            f"{formatted_start_date}-{formatted_until_date}-{uid}.{link_extension}"
-        )
 
-        # Write to a temporary file so we don't overflow the memory
-        with tempfile.NamedTemporaryFile(
-            "w+",
-            prefix=f"playtimereport{formatted_until_date}",
-            suffix=link_extension,
-        ) as temp:
-            # Write the data as a CSV
-            writer = csv.writer(temp)
-            _produce_report(
-                writer,
-                date_label=report_date_label,
-                records=self._fetch_report_records(start=start, until=until),
+        google_drive: GoogleDriveService = self.services.google_drive.palace_internal()
+
+        # create directory hierarchy
+        root_folders = os.environ.get(
+            Configuration.PALACE_GOOGLE_DRIVE_ROOT_ENVIRONMENT_VARIABLE,
+            "palace/no-env-specified",
```

Comment:
minor: This does not seem like a reasonable default. Wouldn't we just want the files to be placed in the root?